lib/authentication/view/edit_profile_page.dart

```diff
@@ -112,9 +287,14 @@ class _EditProfilePageState extends State<EditProfilePage> {
                 User user = snap.data;
                 lastNameController.text = user.lastName;
                 firstNameController.text = user.firstName;
+                if (!authProvider.isVerifiedFromCache) {
+                  emailController.text = authProvider.email.substring(
+                      0, authProvider.email.length - emailDomain.length);
```

Comment:
This is kinda hacky. What if somehow there's a user with a different domain? Maybe try something like `authProvider.email.split('@')[0]`.